json/src/vm/vm.rs

```diff
@@ -35,8 +36,8 @@ pub struct Vm {
 	/// Gas left after transaction execution.
 	#[serde(rename="gas")]
 	pub gas_left: Option<Uint>,
-	/// Logs created during execution of transaction.
-	pub logs: Option<Vec<Log>>,
```

Comment:
isn't this needed for correct functioning of trace RPCs?

Reply:
This is used for tests only. Tracing uses types from `rpc` crate I believe.